.github/workflows/integration-per-language.yml

```diff
@@ -0,0 +1,556 @@
+name: Integration Test for a Single Language
+on:
+  workflow_call:
+    inputs:
+      language:
+        description: "The language to test"
+        required: true
+        type: string
+      repo:
+        description: "The repo to test"
+        required: true
+        type: string
+jobs:
+  helm-dry-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: draft-binary
+      - run: chmod +x ./draft
+      - run: mkdir ./langtest
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          path: ./langtest
+      - name: Execute Dry Run with config file
+        run: |
+          mkdir -p test/temp
+          ./draft --dry-run --dry-run-file test/temp/dry-run.json \
+          create -c ./test/integration/${{inputs.language}}/helm.yaml \
+          -d ./langtest/ --skip-file-detection
+      - name: Validate JSON
+        run: |
+          npm install -g ajv-cli@5.0.0
+          ajv validate -s test/dry_run_schema.json -d test/temp/dry-run.json
+      - name: Execute Dry Run with variables passed through flag
+        run: |
+          mkdir -p test/temp
+          ./draft --dry-run --dry-run-file test/temp/dry-run.json \
+          create \
+          -d ./langtest/ \
+          -l ${{inputs.language}} \
+          --skip-file-detection \
+          --deploy-type helm \
+          --variable PORT=8080 \
+          --variable APPNAME=testingCreateCommand \
+          --variable VERSION=1.11 \
+          --variable BUILDERVERSION=1.11 \
+          --variable SERVICEPORT=8080 \
+          --variable NAMESPACE=testNamespace \
+          --variable IMAGENAME=testImage \
+          --variable IMAGETAG=latest \
+          --variable ENTRYPOINT=myapp.py
+      - name: Validate JSON
+        run: |
+          npm install -g ajv-cli@5.0.0
+          ajv validate -s test/dry_run_schema.json -d test/temp/dry-run.json
```

Comment:
☕️ minor question: we are using ajv as json validation for specific preferred reasons or something else? (Asking only for archival reasons please)

Reply:
there is no preference for ajv outside that it allows validating the schema of our json outputs.

happy to accept PRs for an alternative that better fits the needs of the project